miqa/core/migrations/0014_evaluation.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.7 on 2021-09-20 16:55
```

Comment:
Should we version auto-generated files? Should we give them a better name?

Reply:
Migration files are generated by Django to prep the db tables. My attempts to rename the migration files have only given me errors so far. I'm sure it's possible but not sure how to accomplish it. @zachmullen do you know how?

Reply:
It's common to rename these files, however you'll have to rename them _before_ you apply the migration IIRC. You could also migrate down, rename it, then migrate back up.